styles.css

```diff
@@ -0,0 +1,3 @@
+.header {
+    color: blue;
```

Comment:
- [ ] Also your style is having linters error so doo the same here, add style linter file to fix all issue before sending it for review.